### DISCUSSION TITLE
Vercel CLI behind Proxy

### DISCUSSION THREAD

**Discussion by mitrukahitesh** [vercel/vercel#9424] [Help] [answered]
Do you know if we can use Vercel CLI behind a proxy server? I cannot run CLI commands when I'm behind a proxy.

**Comment by swarnava** [comment]
Hi,

Vercel CLI doesn't support proxy connection for HTTP requests yet. We have an internal ticket (VCCLI-295) where I have added your details.

Feel free to send us a Pull request and we will be more than happy to review. :)

**Reply by mitrukahitesh** [reply]
Sure. Thanks.

**Comment by swarnava** [comment, marked as answer]
Vercel CLI can now use `HTTPS_PROXY` https://github.com/vercel/vercel/pull/9880 :)
